hal/src/nRF52840/exflash_hal.c

```diff
@@ -464,21 +671,22 @@ int hal_exflash_special_command(hal_exflash_special_sector_t sp, hal_exflash_com
         };
 
         if (cmd == HAL_EXFLASH_COMMAND_SLEEP) {
-            cinstr_cfg.opcode = QSPI_MX25_CMD_SLEEP;
-        } else if (cmd == HAL_EXFLASH_COMMAND_WAKEUP) {
-            /* Wake up external flash from deep power down modeby sending read id command */
-            cinstr_cfg.opcode = QSPI_MX25_CMD_READ_ID;
+            cinstr_cfg.opcode = QSPI_STD_CMD_SLEEP;
+        } else if (cmd == HAL_EXFLASH_COMMAND_WAKEUP || cmd == HAL_EXFLASH_COMMAND_READID) {
```

Comment:
Do we want to remove `HAL_EXFLASH_COMMAND_WAKEUP` and make it clear that to wake up the chip we use `HAL_EXFLASH_COMMAND_READID`?